.storage-layouts-normalized/contracts/colony/Colony.sol:Colony.json

```diff
@@ -770,20 +770,9 @@
                 "label": "uint256",
                 "numberOfBytes": "32"
               }
-            },
-            {
-              "contract": "contracts/colony/Colony.sol:Colony",
-              "label": "deprecated",
-              "offset": 0,
-              "slot": "2",
-              "type": {
-                "encoding": "inplace",
-                "label": "bool",
-                "numberOfBytes": "1"
-              }
             }
           ],
-          "numberOfBytes": "96"
+          "numberOfBytes": "64"
```

Comment:
So while `SetExpenditureSingleValues` was missing, these changes are indicating to me that your rebase involving #1280 has gone wrong.